New cmd-let Set-PnPWebPermission to manage permissions of a web

Rus7am opened the pull request:
## Type ##
- [ ] Bug Fix
- [x] New Feature
- [ ] Sample

## Related Issues? ##
Fixes #716

## What is in this Pull Request ? ##
Introduced a new cmd-let **Set-PnPWebPermission** to manage web permissions like **Set-PnPListPermission** for lists.

msftclas commented:
Hi __@Rus7am__, I'm your friendly neighborhood Microsoft Pull Request Bot (You can call me MSBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.microsoft.com.
    </span>

TTYL, MSBOT;

msftgits commented:
Hi, I am closing and re-opening this PR to bump the CLA bot. Sorry for the inconvenience!

msftclas commented:
Hi __@Rus7am__, I'm your friendly neighborhood Microsoft Pull Request Bot (You can call me MSBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, MSBOT;

VesaJuvonen commented:
thx Rustam, we'll process this PR during upcoming days.

erwinvanhunen commented:
Hi! 

I'm processing this PR, and I notice in the help examples that you give an example where you use the Identity parameter, however that parameter is not present in the cmdlet. Can you please fix that? Also, please specify a default parameter set, alike

```csharp
 [Cmdlet(VerbsCommon.Set, "PnPWebPermission", DefaultParameterSetName = "Group")]
```

Rus7am commented:
Hi Erwin!

I'm really sorry, I didn't think about it well enough. I was under the impression that **Identity** and **Url** parameters are handled by the parent **PnPWebCmdlet** class.

I added parameter handling and necessary parameter sets. I also added unit tests.

erwinvanhunen commented:
Thank you!
